src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -361,11 +363,17 @@
                                     mq-endpoint
                                     (command-names command) version payload)))
 
-(defn -main
+(defn- main
```

Comment:
I don't think we want this change. `(defn -main ...)` gives us an external entry point, while `(defn- main ...)` is a ns-private definition.